### PR TITLE
Updates to README.md for clarity

### PR DESCRIPTION
**Pull request by BartleyR**
Updated the `README.md` file as below.

- Removed reference to getting started on NGC (was causing confusion)
- Reworked language on getting started on GitHub
- Minor formatting issues in the Kafka quick start section

### CODE REVIEW

**Review by pdmack — CHANGES_REQUESTED** [review]
"container"

File: README.md [hunk 1]
```diff
@@ -65,16 +55,17 @@ Run the pre-built `runtime` container:
 DOCKER_IMAGE_TAG=22.06-runtime ./docker/run_container_release.sh
```
Comment (pdmack) [inline README.md:55]: This is where people will stub their toes.

"If you are not building the runtime container locally from the current git branch, use an applicable tag from NGC. For example, `DOCKER_IMAGE_TAG=22.04-runtime`.
More official release tags can be found at https://catalog.ngc.nvidia.com/orgs/nvidia/teams/morpheus/containers/morpheus/tags"

File: README.md [hunk 2]
```diff
@@ -99,44 +90,52 @@ Depending on your configuration, it may be necessary to start additional service
    - See the Quick Launch Kafka section.
  - `inf-triton`
    - Requires a running Triton server
-   - See the launching Triton section.
+   - See the Launching Triton Server section.
 
-### Quick-Launch Kafka Cluster
+### Quick Launch Kafka Cluster
```
Comment (pdmack) [inline README.md:95]: All this Kafka docker-compose stuff is a little icky.

**Comment by BartleyR** [comment]
@pdmack Fixed the typo

**Review by BartleyR — COMMENTED** [review]
Fixed the container typo

**Review by pdmack — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (BartleyR) [reply README.md:55]: But do we want a ref to 22.04 in the 22.06 docs?

File: README.md (hunk 2 above)
Comment (BartleyR) [reply README.md:95]: Define "icky"

**Review by pdmack — APPROVED** [review]

**Comment by BartleyR** [comment]
@gpucibot merge

**Pull request merged**